### ISSUE TITLE
minpyver: not erroring on '{}'.format() for python 2.6

### ISSUE THREAD

**Issue by sevagh** [saltstack/salt-pylint#8]
Hello.

`'{}'.format(a)` works in Python 2.7 but not earlier Python (in 2.6, for example, I need to use `'{0}'.format(a)`.

I'm using the minpyver module from this repo to check Python 2.6 compatibility on my codebase, and noticed it does not catch this case.

Should I write a pull request to detect this condition? Was it omitted on purpose?

Thanks.

Some tests.

`{}.format` not working in Python 2.6:
```
Python 2.6.6 (r266:84292, Jan 22 2014, 09:42:36) 
[GCC 4.4.7 20120313 (Red Hat 4.4.7-4)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> string1 = '{}'.format('lol')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: zero length field name in format
>>> string1 = '{0}'.format('lol')
>>> print(string1)
lol
>>> 
```
Pylint config:
```
$ cat setup.cfg 
[MASTER]
load-plugins=saltpylint.minpyver
minimum-python-version=2.6
[MESSAGES CONTROL]
disable=all
enable=E0598
```

Output on testfile:
```
$ cat testfile.py 
if __name__ == '__main__':
    str1 = '{}'.format('lol')
    str2 = '{0}'.format('wat')
$ pylint --rcfile=setup.cfg -e E0598 -r n testfile.py 
$ echo $?
0
```

**Comment by sevagh** [comment]
I have a tentative patch, looks like this:

```
git diff
diff --git a/saltpylint/ext/pyqver2.py b/saltpylint/ext/pyqver2.py
index 4735459..aba8397 100644
--- a/saltpylint/ext/pyqver2.py
+++ b/saltpylint/ext/pyqver2.py
@@ -212,6 +212,8 @@ class NodeChecker(object):
             and isinstance(node.expr.value, str)
             and node.attrname == "format"):
             self.add(node, (2,6), "string literal .format()")
+            if ('{}' in node.expr.value):
+                self.add(node, (2,7), "i don't know")
         self.default(node)
     def visitIfExp(self, node):
         self.add(node, (2,5), "inline if expression")
```

**Comment by s0undt3ch** [comment]
Please see https://github.com/saltstack/salt-pylint/blob/master/saltpylint/strings.py

Although minpyver should probably also catch those...

**Comment by sevagh** [comment]
Thanks, that helped. Should I do a PR to put the behavior in minpyver or just continue using `saltpylint.strings`?

**Comment by s0undt3ch** [comment]
If strings solves your problem, just use it :)
